doc/source/ray-observability/user-guides/debug-apps/general-debugging.rst

```diff
@@ -59,71 +52,63 @@ And I have this code:
 
   @ray.remote
   def check_file():
-    foo_exists = os.path.exists("/tmp/foo.txt")
-    return foo_exists
+      foo_exists = os.path.exists("/tmp/foo.txt")
+      return foo_exists
 
   futures = []
   for _ in range(1000):
-    futures.append(check_file.remote())
+      futures.append(check_file.remote())
 
   print(ray.get(futures))
 
+In this case, you might receive a mixture of True and False. If ``check_file()`` runs on the Head Node or locally, it finds the file; however, on a Worker Node, it doesn't.
 
-then you will get a mix of True and False. If
-``check_file()`` runs on the Head Node, or we're running
-locally it works. But if it runs on a Worker Node, it returns ``False``.
-
-**Expected behavior**: Most people would expect this to either fail or succeed consistently.
-It's the same code after all.
+**Expected behavior:** Users generally expect file references to either work consistently or to reliably fail, rather than behaving inconsistently.
 
-**Fix**
+**Fix:**
 
-- Use only shared paths for such applications -- e.g. if you are using a network file system you can use that, or the files can be on S3.
-- Do not rely on file path consistency.
+— Use only shared file paths for such applications. For example, a network file system or S3 storage can provide the required consistency.
+— Avoid relying on local files to be consistent across machines.
 
 
-
-Placement Groups are not composable
+Placement Groups aren't composable
```

Comment:
I don't understand why this example works. I tried to make a minimal reproduction to improve this section, but couldn't make it work:

```python
import ray
from ray.util.placement_group import PlacementGroupSchedulingStrategy

@ray.remote(scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group="a"))
def outer():
    @ray.remote(scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group=None))
    def inner():
        print("Hello")
        return

    return ray.get([inner.remote() for i in range(10)])


ray.init()
ray.get(outer.remote())
```

Reply:
ask core team?